marathon/adapter_test.go

```diff
@@ -101,6 +112,25 @@ func (c *mockConverter) convertToApp(service *api.Service) *gomarathon.Applicati
 	return args.Get(0).(*gomarathon.Application)
 }
 
+// Mock the Deployer
+type mockDeployer struct {
+	mock.Mock
+	deployStatus status
+}
+
+func (d *mockDeployer) setStatus(s status) {
+	d.deployStatus = s
```

Comment:
I added the test interface. The setStatus is a helper for the mock deployer which still mocks the Deployer interface but has an additional method setStatus which is used only to inject a return status for the mocked DeployGroup group call